README.md

```diff
@@ -1,15 +0,0 @@
-# SendGrid
```

Comment:
love love love that you included a README.md

At SendGrid, our readmes often include steps on how to spin up the service, execute tests, setup the environment, etc... These are really nice to have.